.eslintignore

```diff
@@ -0,0 +1,6 @@
+/.svelte-kit
+/build
+/node_modules
+/scripts
+svelte.config.js
+vite.config.js
```

Comment:
Yeah it's annoying, I described the reason for adding this above - since the tsconfig `included` array can't be extended in sveltekit as it would override the auto-generated config, TS-eslint throws an error on these files because they aren't included in TS, but it is attempting to lint them with type context. I'll see if there is a workaround.

Reply:
Turns out there is a way (https://kit.svelte.dev/docs/configuration#typescript) - I had no idea this existed! Will commit soon.

Reply:
Solved!